src/js/main.js

```diff
@@ -513,6 +515,15 @@ $(function () {
                                     return !(info.energy >= 1 && (CheckAdjacent(x, y) || (info.cells === 0 && build == null && CanBuildHQ(x, y))));
                                 }
                             };
+                            if(build === builds.ROCK && info.cells > 0) {
+                                items.capture = {
+                                    name: "Obsadit pole (⚡10)",
+                                    callback: CaptureCell,
+                                    disabled: function () {
+                                        return !(info.energy >= 10 && CheckAdjacent(x, y));
+                                    }
+                                };
+                            }
```

Comment:
Tohle určitě ne, proč to ve stejné podmínce akorát přepisovat?

Reply:
Nejak som to nepochopil, vedel by si mi to prosím podrobnejšie vysvetliť ?